### PR TITLE
Cleanup tests

### PR DESCRIPTION
**Pull request by garnett**
Cleaned some tests from boilerplate, moved tests fixtures to ```-setUp```

### CODE REVIEW

File: ResearchKitTests/ORKAccessibilityTests.m [hunk 1]
```diff
@@ -39,11 +39,9 @@ @implementation ORKAccessibilityTests
 
 - (void)setUp {
     [super setUp];
-    // Put setup code here. This method is called before the invocation of each test method in the class.
 }
```
Comment (Keithbsmiley) [inline ResearchKitTests/ORKAccessibilityTests.m]: Could probably remove all these calls too, no need to call them if nothing else is being done.

Comment (modocache) [inline ResearchKitTests/ORKAccessibilityTests.m]: +1. Empty overrides add a lot of noise with no benefit.

Comment (garnett) [inline ResearchKitTests/ORKAccessibilityTests.m]: Agree with you, updated the pull-request.

Comment (YuanZhu-apple) [inline ResearchKitTests/ORKAccessibilityTests.m]: Agree to remove empty overrides.

File: ResearchKitTests/ORKHKSampleTests.m [hunk 2]
```diff
@@ -43,46 +46,34 @@ @implementation ORKHKSampleTests
...
     [super setUp];
-    // Put setup code here. This method is called before the invocation of each test method in the class.
-}
-
-- (void)tearDown
-{
-    // Put teardown code here. This method is called after the invocation of each test method in the class.
-    [super tearDown];
+    _d1 = [NSDate dateWithTimeIntervalSinceReferenceDate:0];
+    _d2 = [NSDate dateWithTimeInterval:10 sinceDate:_d1];
```
Comment (modocache) [inline ResearchKitTests/ORKHKSampleTests.m]: This should probably be in `-[ORKHKSampleTests setUp]`, not `-tearDown`. Do the tests still pass with these changes?

Also, IMHO the tests are more readable with `d1` and `d2` local to each test method, not in the `-setUp` method. To each his own, though! :point_up:

Comment (garnett) [inline ResearchKitTests/ORKHKSampleTests.m]: But they are in ```setUp```...

Comment (modocache) [inline ResearchKitTests/ORKHKSampleTests.m]: Oh, whoops! Misread the diff. My bad! :open_mouth:

**Comment by jwe-apple** [comment]
Yeah, actually agree with @modocache on readability. The whitespace / boilerplate cleanup is fine, but let's revert the switch to using ivars.

**Comment by jwe-apple** [comment]
Yep, now it's just whitespace removal and removing unused overrides, looks good.

**Pull request merged**